src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -32,12 +32,8 @@ if(ARM)
                                 # requires estate=32
                                 ${OV_CPU_ARM_TARGET_GENERIC_ARCHS})
 elseif(AARCH64)
-    if(APPLE)
-        # Apple M1 / M2 is assumed
-        set(OV_CPU_ARM_TARGET_ARCH_DEFAULT arm64-v8.2-a)
-    else()
-        set(OV_CPU_ARM_TARGET_ARCH_DEFAULT arm64-v8a)
-    endif()
+    # Apple M1 / M2 or Raspberry 5 is assumed
+    set(OV_CPU_ARM_TARGET_ARCH_DEFAULT arm64-v8.2-a)
```

Comment:
Future ARM Compute Library releases will have an ability to compile universal library which will support fp32, fp16, SVE, SVE2 kernels in the same library.
Based on this, we can compile a single plugin for all our ARM64 targets and don't have to change base arch.

See https://github.com/ARM-software/ComputeLibrary/issues/1053#issuecomment-1845611959